apidoc/Titanium/UI/Window.yml

```diff
@@ -800,6 +800,18 @@ properties:
     since: "3.1.3"
     platforms: [iphone,ipad]
 
+  - name: sustainedPerformanceMode
+    summary: Maintain a sustainable level of performance.
+    description: |
+        Performance can fluctuate dramatically for long-running apps, because the system throttles system-on-chip engines as device components reach their temperature limits. This fluctuation presents a moving target for app developers creating high-performance, long-running apps.
+        
+        Setting this feature to true will set sustained performance mode for the corresponding window. If property is undefined then it defaults to false.
+        
+        Note: feature is only available on supported devices and functionality is experimental and subject to change in future releases. See [Android docs](https://developer.android.com/about/versions/nougat/android-7.0.html#sustained_performance_api) and [Google VR docs](https://developers.google.com/vr/android/ndk/jni/reference/com/google/vr/ndk/base/AndroidCompat.html#setSustainedPerformanceMode(Activity, boolean)) for further info.
+    type: Boolean
```

Comment:
The lines are still too long. Example of proper line breaks:
```yml
Performance can fluctuate dramatically for long-running apps, because the system throttles 
system-on-chip engines as device components reach their temperature limits. This fluctuation 
presents a moving target for app developers creating high-performance, long-running apps.
     
Setting this feature to true will set sustained performance mode for the corresponding window. 
If the property is undefined it defaults to false.
        
Note: feature is only available on supported devices and functionality is experimental and subject 
to change in future releases. See the [Android docs](https://developer.android.com/about/versions/nougat/android-7.0.html#sustained_performance_api) and [Google VR docs](https://developers.google.com/vr/android/ndk/jni/reference/com/google/vr/ndk/base/AndroidCompat.html#setSustainedPerformanceMode(Activity, boolean)) for further info.
```